builder/builder.go

```diff
@@ -504,14 +504,14 @@ func (b *Builder) createFilesForVolume(ctx context.Context, volMount *volume.Vol
 	return nil
 }
 
-func (b *Builder) populateVolumeWithFiles(ctx context.Context, volMount *volume.VolumeMount) error {
+func (b *Builder) populateVolumeWithFiles(ctx context.Context, volMount *volume.Volume) error {
 	var dataContainerArgs []string
 	var dataSB strings.Builder
 	if runtime.GOOS == util.WindowsOS {
 		dataContainerArgs = []string{"powershell.exe", "-Command"}
-		dataSB.WriteString("docker run --rm -v " + b.workspaceDir + "\\" + volMount.Name + ":c:\\source -v ")
+		dataSB.WriteString("docker run --rm -v " + b.workspaceDir + ":c:\\source -v ")
```

Comment:
-w c:\source
?

Reply:
It seems to work without that as well locally. Do you think I should change it anyways?